packages/react-headless/src/polyfills/mitt-module-cjs.cts

```diff
@@ -0,0 +1,2 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+export const mitt = require('mitt');
```

Comment:
I saw in other places that you added two file polyfills, but what is the `...-modue-cjs.cts` file actually used for? Isn't only the `...-module.ts` file needed?